builtin/game/register.lua

```diff
@@ -338,25 +398,24 @@ function core.run_callbacks(callbacks, mode, ...)
 	if cb_len == 0 then
 		if mode == 2 or mode == 3 then
 			return true
-		elseif mode == 4 or mode == 5 then
+		end
```

Comment:
On return it 'jumps' to anothre line and continues with the code execution there, renaming `elseif` to `if` has no effect.